src/buckets/components/createBucketForm/MeasurementSchemaSection.tsx

```diff
@@ -469,17 +469,79 @@ export const MeasurementSchemaSection: FC<Props> = ({
     'https://docs.influxdata.com/influxdb/cloud/organizations/buckets/bucket-schema/'
 
   const schemas = measurementSchemaList?.measurementSchemas
-  let readPanels = null
+  let readSection = null
   if (schemas) {
-    readPanels = schemas.map((oneSchema, index) => (
+    /**
+     * typescript transpiler needs this; using just setDownloadType
+     * yields an error (claims the argument is of type string)
+     * */
+    const onDownloadTypeChange = (dtype: DownloadTypes) => {
+      setDownloadType(dtype)
+    }
+
+    const readPanels = schemas.map((oneSchema, index) => (
       <EditingPanel
         key={`mss-ep-${index}`}
         measurementSchema={oneSchema}
         index={index}
         onAddUpdate={onAddUpdate}
         toggleUpdate={toggleUpdate}
+        downloadFormat={downloadType}
       />
     ))
+    const downloadToggle = (
+      <FlexBox direction={FlexDirection.Row}>
+        <InputLabel style={{marginRight: 10}}>
```

Comment:
please create a static variable to hold `{marginRight: 10}` to prevent unnecessary re-renders.

https://www.debugbear.com/blog/react-rerenders#passing-objects-as-props